components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/interceptors/eTag/ETagOutInterceptor.java

```diff
@@ -43,16 +42,16 @@ public ETagOutInterceptor() {
     public void handleMessage(Message message) throws Fault {
         if (checkETagSkipList(message.getExchange().getInMessage().get(Message.PATH_INFO).toString(),
                 message.getExchange().getInMessage().get(Message.HTTP_REQUEST_METHOD).toString())) {
-            log.info("Skipping ETagOutInterceptor for URI : " + message.getExchange().getInMessage().get(Message.PATH_INFO).toString());
+            if (log.isDebugEnabled()){
+                log.debug("Skipping ETagOutInterceptor for URI : " + message.getExchange().getInMessage().get(Message.PATH_INFO).toString());
+            }
             return;
 
         }
         MultivaluedMap<String, Object> headers = (MetadataMap<String, Object>) message.get(Message.PROTOCOL_HEADERS);
         if (headers == null) {
             headers = new MetadataMap<>();
         }
-        MessageContext currentMessageContext = MessageContext.getCurrentMessageContext();
```

Comment:
Removed it since there were no usages as we no longer use message contexts to send the ETag value to the ETagOutInterceptor